README.md

```diff
@@ -1,2 +1,83 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # budget-planner
-This helps users to plan several budgets which can be grouped into categories, User can create new categories of which he/she can make a budget plan
+
+> This helps users to plan several budgets which can be grouped into categories, Users can create new categories of which they can make a budget plan, by clicking on a category a user can create a new item with the amount they want to allocate for that item.
+
+Mobile Landing page                      |  Mobile sign up
+:---------------------------------------:|:---------------------------------------:
+![](./public-img/Landing.PNG)            |  ![](./public-img/signup.PNG)
+
+Mobile Categories page                   |  Mobile Items page
+:---------------------------------------:|:---------------------------------------:
+![](./public-img/category-page.PNG)      |  ![](./public-img/items-page.PNG)
+
+
+## Built With
+
+- Ruby
+- Rails
+
+# Pre-requirements
+
+- Vscode/Atom
+- Ruby
+- Rails
+- Nodejs
+- Yarn
+- Sqlite3
+- Postgresql
+- Git
+
+## demo
+
+[heroku](https://immense-bastion-80345.herokuapp.com/)
```

Comment:
- [ ] Please update your live demo link with the latest updates of your app because it crushes once I sign up! 🙏